webhook.rst

```diff
@@ -84,8 +84,14 @@ Mailer service  Parser service name
 =============== ==========================================
 Mailgun         ``mailer.webhook.request_parser.mailgun``
 Postmark        ``mailer.webhook.request_parser.postmark``
+Sendgrid        ``mailer.webhook.request_parser.sendgrid``
 =============== ==========================================
 
```

Comment:
We should add a versionadded for Mailgun and Postmark too

Reply:
As a third column, perhaps? To have a bit better overview when more are added in the future?

Reply:
No we usually remove the versionadded directives when merging up to a new major, so we should stick with this

Reply:
The Webhook itself is covered by a 6.3 versionadded in the top of the document.

Reply:
Lets do this in the PR of Alex #19268